packages/runner-ct/src/lib/ResizableBox.tsx

```diff
@@ -24,7 +24,9 @@ export const ResizableBox: React.FC<ResizableBoxProps> = ({
   const startClientXRef = React.useRef(width)
   const resizingRef = React.useRef<HTMLDivElement>(null)
 
-  const handleResize = React.useCallback((e) => {
+  const handleResize = React.useCallback((e: MouseEvent) => {
+    e.preventDefault()
```

Comment:
I see, that could be better. 
Did you observe any change in performance?

Reply:
If you have not already done so, please make sure this works the same in all 3 supported browser FF, Edge and Chrome

Reply:
I didn't, but this approach also prevents text selection inside testing iframe

Reply:
After thought: No need to test edge it will use the same engine as chrome.
I will test FF right now and report back

Reply:
https://user-images.githubusercontent.com/5592465/109038897-d58c2880-7691-11eb-8c90-6b00b5161fcb.mov

Dang it !!!

Reply:
@dmtrKovalenko
What I would do here is remove the user-select changes from here so that we can merge this bug fix and deal with the issue of selecting the iframe in another ticket or PR.

Reply:
May need `stopPropogation` for FF

Reply:
Fixed this for all the browsers in a much better way. Thanks, folks for this comment!